daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/EvDelimiters.scala

```diff
@@ -68,6 +68,13 @@ abstract class DelimiterParseEv(delimType: DelimiterTextType.Type, override val
     if (converterResult.length == 1 && converterResult(0) == "") {
       Array()
     } else {
+      System.out.println("In DelimiterParseEv: " + expr)
+      System.out.println("converterResult: " + converterResult)
+      if (converterResult.contains("")) {
+        System.out.println("Empty delimiter found!!!")
```

Comment:
When running testOnly org.apache.daffodil.section12.delimiter_properties.TestDelimiterProperties this print statement (the one before the SDE call) gets displayed, but not the one after the SDE.  However, it continues processing til it hits the Assert call